File: support_tools/swigwin-3.0.12/Examples/test-suite/go/overload_extend2_runme.go
package main

import "./overload_extend2"

func main() {
	f := overload_extend2.NewFoo()
	if f.Test(3) != 1 {
		panic(0)
	}
	if f.Test("hello") != 2 {
		panic(0)
	}
	if f.Test(3.5, 2.5) != 3 {
		panic(0)
	}
	if f.Test("hello", 20) != 1020 {
		panic(0)
	}
	if f.Test("hello", 20, 100) != 120 {
		panic(0)
	}

	// C default args
	if f.Test(f) != 30 {
		panic(0)
	}
	if f.Test(f, 100) != 120 {
		panic(0)
	}
	if f.Test(f, 100, 200) != 300 {
		panic(0)
	}
}

File: support_tools/swigwin-3.0.12/Examples/test-suite/octave/overload_extend2_runme.m
overload_extend2

f = overload_extend2.Foo();
if (f.test(3) != 1)
    error
endif
if (f.test("hello") != 2)
    error
endif
if (f.test(3.5,2.5) != 3)
    error
endif
if (f.test("hello",20) != 1020)
    error
endif
if (f.test("hello",20,100) != 120)
    error
endif

# C default args
if (f.test(f) != 30)
    error
endif
if (f.test(f,100) != 120)
    error
endif
if (f.test(f,100,200) != 300)
    error
endif

